packages/turbo-ignore/src/index.ts

```diff
@@ -25,6 +25,11 @@ if (!scope) {
   console.error(
     "Error: app scope not found. turbo-ignore inferencing failed, proceeding with build."
   );
+  if (!pathScope.scope) {
+    console.error(
+      "Hint: turbo-ignore cannot process because the monorepo root package.json does not have the name field.\nset or pass it as an explicit argument to turbo-ignore"
```

Comment:
```suggestion
      "Error: the root package.json is missing the \"name\" field.\nSet this field or pass "scope" as an explicit argument to turbo-ignore."
```

side question: what exactly would be passed as an explicit arg?

Reply:
@mehulkar 
thank you for your suggestion. I commit your suggestion code and refactor it a little. 

5b14d27
6da7a20
edddb8c

> side question: what exactly would be passed as an explicit arg?

I read in the code that turbo-ignore is supposed to process with the value of the name field in package.json as an arg.
Finally, this argument is supposed to be passed to `turbo` command  as a filter arg.

So I think the following use case might be possible.

### 1. package.json has the name field ("name": "hoge")

```bash
cd ./packages/hoge
npx turbo-ignore
≫ Using Turborepo to determine if this project is affected by the commit...
≫ Inferred `hoge` as scope from "/Users/t-i-0414/project/packages/hoge/package.json"
≫ Analyzing results of `npx turbo run build --filter=hoge...[HEAD^] --dry=json`...
≫ This project and its dependencies are not affected
≫ Ignoring the change
```

### 2. package.json does not have the name field

```bash
cd ./packages/hoge
npx turbo-ignore  
≫ Using Turborepo to determine if this project is affected by the commit...
Error: app scope not found. turbo-ignore inferencing failed, proceeding with build.
```

### 3. package.json does not have the name field but passing an arg

```bash
cd ./packages/hoge
npx turbo-ignore hoge
≫ Using Turborepo to determine if this project is affected by the commit...
≫ Inferred `hoge` as scope from arguments
≫ Analyzing results of `npx turbo run build --filter=hoge...[HEAD^] --dry=json`...
≫ This project and its dependencies are not affected
≫ Ignoring the change
```

### 4. package.json has the name field (name: hoge) and passing an arg fuga

```bash
cd ./packages/hoge
npx turbo-ignore fuga
≫ Using Turborepo to determine if this project is affected by the commit...
≫ Inferred `fuga` as scope from arguments
≫ Analyzing results of `npx turbo run build --filter=fuga...[HEAD^] --dry=json`...
≫ This project and its dependencies are not affected
≫ Ignoring the change
```

I assume that this PR will take care of case 2 as following.
↓
```
cd ./packages/hoge
npx turbo-ignore
≫ Using Turborepo to determine if this project is affected by the commit...
Error: app scope not found. turbo-ignore inferencing failed, proceeding with build.
Error: the package.json is missing the "name" field.
Set this field or pass "scope" as an explicit argument to turbo-ignore.
```